.github/actions/isStagingDeployLocked/isStagingDeployLocked.js

```diff
@@ -0,0 +1,14 @@
+const _ = require('underscore');
+const core = require('@actions/core');
+const github = require('@actions/github');
+const GithubUtils = require('../../libs/GithubUtils');
+
+const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
+const githubUtils = new GithubUtils(octokit);
+
+githubUtils.getStagingDeployCash()
+    .then(({labels}) => core.setOutput('IS_LOCKED', _.contains(_.pluck(labels, 'name'), '🔐 LockCashDeploys 🔐')))
```

Comment:
Not totally required to switch, but why make this it's own action when we could use something already built like: https://github.com/marketplace/actions/github-api-request ?

Reply:
Ah is it because this needs `getStagingDeployCash()`?

Reply:
Yeah, it's just to take advantage of the code we've already built for fetching the open StagingDeployCash